client/i18n/locales/english/intro.json

```
@@ -2052,7 +2052,10 @@
       "yshh": { "title": "237", "intro": [] },
       "wyxz": { "title": "238", "intro": [] },
       "dtfv": { "title": "239", "intro": [] },
-      "miza": { "title": "240", "intro": [] },
+      "lab-quick-sort-algorithm": {
```

```suggestion
      "lab-quicksort-algorithm": {
```
It's one word. https://dl.acm.org/doi/10.1145/366622.366644

---

No worries. I'm not sure it's worth the time for the old stuff since it sees so little use.